src/test/java/de/tum/in/www1/artemis/AccountResourceIntegrationTest.java

```
@@ -50,27 +88,129 @@ public void registerAccount() throws Exception {
         request.postWithoutLocation("/api/register", userVM, HttpStatus.CREATED, null);
     }
 
+    @Test
+    public void registerAccountTooLongPassword() throws Exception {
+        // setup user
+        User user = ModelFactory.generateActivatedUser("ab123cd");
+        ManagedUserVM userVM = new ManagedUserVM(user);
+        userVM.setPassword("e".repeat(ManagedUserVM.PASSWORD_MAX_LENGTH + 1));
+
+        // make request
+        request.postWithoutLocation("/api/register", userVM, HttpStatus.BAD_REQUEST, null);
+    }
+
+    @Test
+    public void registerAccountTooShortPassword() throws Exception {
+        // setup user
+        User user = ModelFactory.generateActivatedUser("ab123cd");
+        ManagedUserVM userVM = new ManagedUserVM(user);
+        userVM.setPassword("e".repeat(Math.max(0, ManagedUserVM.PASSWORD_MIN_LENGTH - 1)));
```

Imo you could also throw an exception if the min length is less than 0